library/common/config/config.cc

```diff
@@ -387,6 +387,12 @@ stats_sinks: *stats_sinks
         - safe_regex:
             google_re2: {}
             regex: '^cluster\.[\w]+?\.upstream_rq_[\w]+'
+        - safe_regex:
+            google_re2: {}
```

Comment:
Is there ever a situation we'd want to use re2?

Reply:
Frankly speaking - I do not know. I was an ignorant and followed the established pattern + I'm just more familiar with standard regexes so felt more comfortable doing that.